cvereporter/nist_enhance.py

```diff
@@ -0,0 +1,88 @@
+from cyclonedx.model.impact_analysis import ImpactAnalysisAffectedStatus
+from cyclonedx.model.vulnerability import Vulnerability, VulnerabilitySource,VulnerabilityScoreSource, VulnerabilityRating, VulnerabilitySeverity, BomTarget, BomTargetVersionRange
+import requests
+import json
+
+"""
+this file has the utilities for downloading data about cves from NIST and updating Vulnerability objects with the data
+"""
+def fetch_nist(url: str, id: str) -> dict:
```

Comment:
Would be good to separate the def functions with some whitespace

Reply:
i'll run  a formatter

Reply:
ran the formatter, added instructions for how to run